rpc/block_test.go

```diff
@@ -134,7 +134,7 @@ func TestBlockWithTxHashes(t *testing.T) {
 
 	type testSetType struct {
 		BlockID                          BlockID
-		ExpectedError                    *RPCError
+		ExpectedError                    error
```

Comment:
nit: rename it to `ExpectedErr` since in other tests type that looks like this one has this naming